src/cluecoins/database.py

```diff
@@ -52,12 +52,14 @@ def update_transaction(conn: Connection, id_: int, rate: Decimal, amount: Decima
     )
 
 
-def iter_accounts(conn: Connection) -> Iterator[tuple[int, str, Decimal]]:
+def iter_accounts(
+    conn: Connection, old_currency: str = 'USDT', new_currency: str = 'USD'
+) -> Iterator[tuple[int, str, Decimal]]:
     for row in conn.cursor().execute(
         'SELECT accountsTableID, accountCurrency, accountConversionRateNew FROM ACCOUNTSTABLE;'
     ):
         id_, currency, rate = row
-        currency = currency.replace('USDT', 'USD')
+        currency = currency.replace(old_currency, new_currency)
```

Comment:
I don't think we need to perform substitution in `iter_accounts` method itself. A separate method to replace by subs.table?

Reply:
Task #36